ci/autoscaler/scripts/deploy-apps.sh

```diff
@@ -13,6 +13,12 @@ popd > /dev/null
 function deploy() {
   log "Deploying autoscaler apps for bosh deployment '${deployment_name}' "
```

Comment:
shouldn't be only "Deploying autoscaler apps" ?

Reply:
Since, it is partial vm and apps setup at this points, we can have it like it is - clarifeid